README.md

```diff
@@ -33,7 +33,9 @@ pip list --outdated --format=freeze | grep -v '^\-e' | cut -d = -f 1  | xargs -n
 
 ### Run FlyFlix
 
-Once installed, you should be able to run FlyFlix by executing a python script on the terminal: `python flyflix.py`. Once the application is up and running, the client can be started by pointing a web browser to an URL with the IP address of the server at port 17000, for example <http://127.0.0.1:17000> or <http://192.168.1.27:17000>. If you are unsure what your IP address is, you might be able to see it through the terminal command `ip addr` (Mac and Linux) or `ifconfig` (Windows). On Linux, you should be able to start FlyFlix through `make localhost` instead of the direct call to the python script, which will also show you the currently used address.
+Once installed, you should be able to run FlyFlix by executing a python script on the terminal: `python flyflix.py`. Once the application is up and running, the client can be started by pointing a web browser to an URL with the IP address of the server at port 17000, for example <http://127.0.0.1:17000> or <http://192.168.1.27:17000>. If you are unsure what your IP address is, you might be able to see it through the terminal command `ip addr` (Mac and Linux) or `ipconfig` (Windows). On Linux, you should be able to start FlyFlix through `make localhost` instead of the direct call to the python script, which will also show you the currently used address.
```

Comment:
Maybe it makes sense to distinguish between running the browser on the same computer and on a remote tablet? If you run the browser on the same computer, http://127.0.0.1:17000 will always work. Only if you want to see the stimulus from a remove client, for example a tablet, you will need the `ipconfig` or `ip addr` command. For both, Linux and Windows, you would need to attach port :17000 to the address.

Does that explanation make sense to you? Would you be able to change the documentation accordingly? You could also let me do another draft and then edit it afterwards? What do you prefer?